tests/integration/relations/test_opensearch_provider.py

```diff
@@ -89,8 +91,11 @@ async def test_create_relation(ops_test: OpsTest, application_charm, opensearch_
         timeout=1600,
         status="active",
     )
+    # Return all update-status-hook-interval to 5m, as all idle_period values are set to 70s
+    await ops_test.model.set_config({"update-status-hook-interval": "5m"})
```

Comment:
hi @Mehdi-Bendriss but then, we have to set all idle_period to less than 1m (e.g. 50s). Otherwise, juju will check status and you will see the latest refresh data being always smaller than 70s (where update-status runs at 60s)